microsoft/react-native-macos
# Fixed Images not announcing in VoiceOver

#### Please select one of the following

## Summary

Images were not being announced as Images by Voiceover. This is unexpected behavior and would have caused visually impaired users to miss images in our experience.

## Changelog

[MacOS] [Fixed] - Fixed Voiceover reading of Image 

## Test Plan

1. Go into RNTester/js/examples/Image/ImageExample.js and add accessible={true} accessibilityLabel="foo" to 'Plain Network Image' on line 415.
2. Compile and run app.
3. Navigate to Image page
4. Turn on Voiceover.
5. Navigate to the Plain Network Image.
- You should hear/see voiceover announce "foo, image"

 
![Screen Shot 2020-06-26 at 14 34 36](https://user-images.githubusercontent.com/65255457/85903083-5ec52580-b7ba-11ea-8365-acd468e39fa0.png)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-macos/pull/475)
